File: plugins/tene/test/fixtures/sample.go
package payments

import (
	"fmt"
	pay "github.com/acme/pay"
)

// func fakeInComment() {}
var raw = `func alsoFake() {}`

func ProcessPayment(input Input) (Result, error) {
	r, err := chargeCard(input)
	if err != nil {
		return recordFailure(input, err)
	}
	return r, nil
}

func (s *PaymentService) Charge(amount int) error {
	return s.gateway.Send(amount)
}

type Input struct{ Amount int }
type Gateway interface{ Send(int) error }

File: plugins/tene/test/fixtures/sample.py
from .payments import charge_card
import logging as log

# def fake_in_comment(): pass
"""
class FakeDoc: pass
"""
s = "def also_fake(): pass"

def process_payment(input):
    result = charge_card(input)
    if not result.ok:
        return record_failure(input, result)
    return result

class PaymentService:
    def charge(self, amount):
        return self.gateway.send(amount)

    def _private_helper(self):
        pass

handler = getattr(obj, name)
